### DISCUSSION TITLE
Woche 2 (Semster 4) - RMMM

### DISCUSSION THREAD

**Discussion by lukashert** [Chrissi-Ruege/HealthKeeper#22] [General]
Beschreibungen der einzelnen Spalten in der RMMM-Tabelle (falls notwendig, aber gerade für die Spalten probability und so wo nur zahlen drin stehen, doch ganz praktisch. Kann man ja mit in den Blog Beitrag nehmen um die Tabelle zu beschreiben):

1. **Risk ID:** Eine eindeutige Kennung oder Nummer, die jedem identifizierten Risiko zugeordnet wird. Dies ermöglicht eine einfache Referenzierung und Verfolgung der Risiken im Projekt.

2. **Category:** Die Kategorie oder Art des Risikos, z.B. technische Risiken, zeitliche Risiken, Sicherheitsrisiken oder Risiken im Zusammenhang mit den Benutzern.

3. **Risk Description:** Eine klare Beschreibung des identifizierten Risikos, einschließlich möglicher Ursachen und potenzieller Auswirkungen auf das Projekt.

4. **Probability:** Die Wahrscheinlichkeit, dass das Risiko eintritt, normalerweise auf einer Skala von beispielsweise 1 bis 5, wobei 1 eine geringe Wahrscheinlichkeit und 5 eine hohe Wahrscheinlichkeit darstellt.

5. **Impact:** Die potenziellen Auswirkungen des Risikos auf das Projekt, falls es eintritt, oft bewertet auf einer Skala von beispielsweise 1 bis 5, wobei 1 eine geringe Auswirkung und 5 eine hohe Auswirkung darstellt.

6. **Risk Score:** Eine berechnete Bewertung des Risikos, die sich aus der Multiplikation von Wahrscheinlichkeit und Auswirkung ergibt. Es hilft bei der Priorisierung der Risiken basierend auf ihrer potenziellen Schwere.

7. **Mitigation Strategy:** Die geplante Strategie oder die Maßnahmen, die ergriffen werden, um das Risiko zu mindern oder zu eliminieren.

8. **Indicator:** Anzeichen oder Warnsignale, die darauf hinweisen können, dass das Risiko eintritt oder sich verschlimmert. Diese helfen bei der Überwachung des Risikos im Verlauf des Projekts.

9. **Contingency Plan:** Ein Plan für den Fall, dass das Risiko eintritt. Dies umfasst typischerweise Schritte, die unternommen werden, um die Auswirkungen des Risikos zu mindern oder das Projekt wieder auf Kurs zu bringen.

10. **Responsible:** Die Person oder Gruppe, die für die Überwachung und die Umsetzung der Risikominderungsmaßnahmen verantwortlich ist.

11. **Status:** Der aktuelle Status des Risikos, einschließlich Maßnahmen, die bereits ergriffen wurden, und deren Wirksamkeit.

12. **Last Modified Date:** Das Datum, an dem die Informationen in der Zeile zuletzt aktualisiert wurden, um sicherzustellen, dass die Tabelle stets aktuell ist.

RMMM-Tabelle mit den Risiken R1 bis R5:

| Risk ID | Category | Risk Description                                                                                   | Probability | Impact | Risk Score | Mitigation Strategy                                                                                             | Indicator                                                      | Contingency Plan                                                 | Responsible | Status | Last Modified Date |
|---------|----------|---------------------------------------------------------------------------------------------------|-------------|--------|------------|------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------|------------------------------------------------------------------|-------------|--------|--------------------|
| R1      | Technisch | Datenverlust oder -korruption aufgrund unzureichender Datensicherung                              | 4           | 5      | 20         | Regelmäßige automatische Backups implementieren                                                                   | Überwachung der Backup-Integrität und Testwiederherstellung    | Schnelle Wiederherstellung aus dem letzten Backup                  | Backend-Team     |        |                    |
| R2      | Zeitlich  | Verzögerungen bei der Entwicklung aufgrund unerwarteter Komplexität oder Ressourcenmangel        | 3           | 4      | 12         | Implementierung von Agilen Methoden zur flexiblen Anpassung des Projektumfangs                                      | Fortschrittsberichte, um Verzögerungen frühzeitig zu erkennen   | Anpassung des Zeitplans und der Ressourcenallokation               | Teammitglieder |        |                    |
| R3      | Sicherheit | Datenschutzverletzung aufgrund von Schwachstellen in der Anmeldungs- oder Datenspeicherungsfunktion | 2           | 4      | 8          | Implementierung von Verschlüsselungstechniken für Benutzerdaten und Zugriffskontrollen                           | Überwachung von Anmeldeversuchen und ungewöhnlichem Datenzugriff | Sofortige Schließung von Sicherheitslücken und Benachrichtigung | Teammitglieder |        |                    |
| R4      | Nutzer    | Mangelnde Benutzerakzeptanz aufgrund einer unzureichenden Benutzeroberfläche oder Funktionalität   | 3           | 3      | 9          | Durchführung von Benutzerumfragen und -tests während des Entwicklungsprozesses, um Feedback zu erhalten        | Rückmeldungen von Benutzern über die Plattformnutzung           | Aktualisierung der Benutzeroberfläche oder -funktionen           | Frontend-Team     |        |                  |
| R5      | Zeitlich  | Reduzierte Entwicklungsressourcen aufgrund der Verkleinerung des Teams von vier auf weniger Mitglieder, was zu Zeitdruck und Priorisierung von Aufgaben führt                      | 4           | 4      | 16         | Überprüfung und Anpassung des Projektumfangs   | Fortschrittsberichte über den Fortschritt und die Qualität | Umschichtung von Ressourcen und Aufgabenpriorisierung bei Verzögerungen oder Engpässen                           | Teammitglieder |        |                    |

Risiko R5 ist eingetreten:

**Mitigation Strategy:**
Die Anpassung des Projektumfangs ist eine zentrale Maßnahmen, um auf die reduzierten Ressourcen zu reagieren. Dies bedeutet, dass wir den Umfang des Projekts überprüfen und gegebenenfalls reduzieren müssen, um sicherzustellen, dass die verbleibenden Aufgaben innerhalb der verfügbaren Zeit und mit den vorhandenen Ressourcen bewältigt werden können. 

**Contingency Plan:**
Im Falle von Verzögerungen oder Engpässen greift der Contingency Plan des Projektmanagements. Dieser Plan sieht vor, dass das Projektmanagement bei Bedarf Ressourcen umschichtet und Aufgabenprioritäten neu setzt. Dies kann bedeuten, dass weniger kritische Aufgaben zurückgestellt oder delegiert werden, während die wichtigen Aspekte des Projekts priorisiert werden. Es ist wichtig, dass regelmäßig der Projektfortschritt überwacht wird und frühzeitig potenzielle Verzögerungen erkannt werden. Auf dieser Grundlage können dann rechtzeitig Anpassungen vorgenommen werden, um sicherzustellen, dass das Projekt trotz der Herausforderungen erfolgreich abgeschlossen wird.

Durch eine sorgfältige Planung, flexible Anpassung und proaktive Überwachung können wir effektiv auf die reduzierten Ressourcen reagieren und sicherstellen, dass das Projekt seine Ziele erreicht, auch wenn oben genannte oder weitere nicht aufgeführte Risiken eintreten.

**Comment by DossohMael** [comment]
Guter Blogbeitrag. 
Mir gefällt vor allem die ausführliche Erklärung der Einzelteile eures RMMM's.
